underfs/gsg/src/main/java/alluxio/underfs/gsg/GSGUnderFileSystem.java

```diff
@@ -79,7 +81,7 @@ public static GSGUnderFileSystem createInstance(AlluxioURI uri, UnderFileSystemC
     String bucketName = UnderFileSystemUtils.getBucketName(uri);
     GoogleCredentials credentials;
     if (conf.containsKey(GSGPropertyKey.GSG_CREDENTIAL_PATH)) {
-      LOG.info("Create GSGUnderFileSystem with UnderFileSystemConfiguration conf");
+      LOG.info("Create GSGUnderFileSystem with UnderFileSystemConfiguration");
```

Comment:
not related to the mode change but why do we need to check both ufs and global conf? ufs conf already falls back on global conf

Reply:
Removed the global conf get property, thanks